content/courses/mobile/solana-mobile-dapps-with-expo.md

```diff
@@ -1533,7 +1598,7 @@ approve the app. Fetch all of the NFTs by tapping `Fetch NFTs`. Lastly, tap
 Congratulations! That was not an easy or quick lab. You're doing great if you've
 made it this far. If you run into any issues, please go back through the lab
 and/or reference the final solution code on the
-[`main` branch in Github](https://github.com/Unboxed-Software/solana-advance-mobile).
+[`main` branch in Github](https://github.com/XuananLe/solana-advance-mobile).
```

Comment:
Moved to https://github.com/solana-developers/mobile-apps-with-expo

Please add a README!